api/swagger.yml

```diff
@@ -3785,6 +3843,128 @@ paths:
         default:
           description: internal server error
 
+  /repositories/{repository}/branches/{branch}/staging/pmpu:
+    parameters:
+      - in: path
+        name: repository
+        required: true
+        schema:
+          type: string
+      - in: path
+        name: branch
+        required: true
+        schema:
+          type: string
+      - in: query
+        name: path
+        description: relative to the branch
+        required: true
+        schema:
+          type: string
+      - in: query
+        name: parts
+        description: number of presigned URL parts required to upload
+        schema:
+          type: integer
+    post:
+      tags:
+        - experimental
+      operationId: createPresignMultipartUpload
+      summary: Initiate a multipart upload
+      description: |
+        Initiates a multipart upload and returns an upload ID with presigned URLs for each part (optional).
+        Part numbers starts with 1. Each part except the last one has minimum size depends on the underlying blockstore implementation.
+        For example working with S3 blockstore, minimum size is 5MB (excluding the last part).
+      responses:
```

Comment:
Added the missing 400 status code for all routes.
The SDK will fail the request in case of bad request.
The number of parts for create is optional, in order to support additional API to provide presigned parts URL request for streaming.